Modules/Scripted/DICOMPlugins/DICOMScalarVolumePlugin.py

```diff
@@ -224,10 +237,34 @@ def examineFiles(self, files):
             for tag in subseriesTags:
                 value = slicer.dicomDatabase.fileValue(file, self.tags[tag])
                 value = value.replace(",", "_")  # remove commas so it can be used as an index
+
                 if tag not in subseriesValues:
                     subseriesValues[tag] = []
-                if not subseriesValues[tag].__contains__(value):
+
+                if tag in vectorTags:
+                    if value != "":
+                        vector = numpy.array([float(element) for element in value.split("\\")])
+
+                        found = False
+                        for subseriesValue in subseriesValues[tag]:
+                            subseriesVector = numpy.array([float(element) for element in subseriesValue.split("\\")])
+
+                            # The 'eps' value default is set to 1.e-6, which reproduces the same behavior as
+                            # the ITK numeric comparison logic (i.e. absolute difference).
+                            # Reference:
+                            #   Class: ITK/Modules/Numerics/Optimizersv4/include/itkObjectToObjectMetric.hxx
+                            #   Method: VerifyDisplacementFieldSizeAndPhysicalSpace
+                            #   URL: https://github.com/hinerm/ITK/blob/master/Modules/Numerics/Optimizersv4/include/itkObjectToObjectMetric.hxx#L471-L475.
+                            if numpy.allclose(vector, subseriesVector, rtol = 0., atol = eps):
```

Comment:
```suggestion
                            if numpy.allclose(vector, subseriesVector, rtol=0.0, atol=eps):
```

While we are not yet using "ruff format" (or black), this will be the recommended style.